src/validation/validation.cpp

```diff
@@ -1743,6 +1751,14 @@ uint32_t GetBlockScriptFlags(const CBlockIndex *pindex, const Consensus::Params
         flags |= SCRIPT_VERIFY_NULLFAIL;
     }
 
+    // The Nov 15, 2018 HF enable sig push only and atrt enforcing also
+    // clean stack rules (see  BIP 62 for more details)
+    if (IsNov152018Enabled(consensusparams, pindex->pprev))
```

Comment:
This is missing the IsNov152018Scheduled() call, which makes it inconsistent with other uses in this PR.  Its actually correct code because if ...Scheduled() returns false ...Enabled() will always return false.  But I think that for clarity we should add it here to make it consistent with the other uses.

But FYI you only really need to use both Scheduled and Enabled if you are checking whether you are before a scheduled fork.  For example, during a rollback you might need to check if the fork was on but is now off.  Another use is to override a configuration "tweak" (if you had one) to the pre or post-fork state based on the fork time.  You only want to override if the fork is actually scheduled.